servers/v3-proxy-api/src/test/fixtures/fetch.ts

```diff
@@ -107,11 +107,11 @@ export const expectedFetch: FetchResponse = {
       },
       tags: {
         research: {
-          item_id: '11231399273',
+          item_id: '2111834840',
```

Comment:
for some reason this was set to be the resolved_id, but it should definitely be the item_id...?